src/plone/testing/z2.py

```diff
@@ -543,6 +543,8 @@ def setUpDatabase(self):
         self['zodbDB'] = zodb.stackDemoStorage(
             self.get('zodbDB'),
             name='Startup')
+        import transaction
```

Comment:
I think thin import can be moved to the top of the file.
There is already an import from OFS so I am quite sure that `transaction` will be available, too when this module is imported.